setup.py

```diff
@@ -0,0 +1,113 @@
+##
+# Copyright 2012-2024 Ghent University
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/easybuilders/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+
+"""
+This script can be used to install easybuild-easyconfigs-archive, e.g. using:
+   easy_install --user .
+ or
+   python setup.py --prefix=$HOME/easybuild
+
+@author: Kenneth Hoste (Ghent University)
+"""
+
+import os
+from distutils import log
+from distutils.core import setup
+
+# note: release candidates should be versioned as a pre-release, e.g. "1.1rc1"
+# 1.1-rc1 would indicate a post-release, i.e., and update of 1.1, so beware!
+#
+# important note: dev versions should follow the 'X.Y.Z.dev0' format
+# see https://www.python.org/dev/peps/pep-0440/#developmental-releases
+# recent setuptools versions will *TRANSFORM* something like 'X.Y.Zdev' into 'X.Y.Z.dev0', with a warning like
+#   UserWarning: Normalizing '2.4.0dev' to '2.4.0.dev0'
+# This causes problems further up the dependency chain...
+VERSION = '5.0.0.dev0'
```

Comment:
@lexming Hmm, we should do something different here I think?

Or do we always want to publish a `x.y.z` version of the `easybuild-easyconfigs-version` package when EasyBuild `x.y.z` is released (I would say no).

Reply:
yeah, there is probably no need for _revisions_, so we just do _major.minor_ then. But I would still keep the _major.minor_ in sync with the releases of the rest of the ecosystem, making releases of the archive is basically one click.